src/classes/todo-item.js

```diff
@@ -0,0 +1,10 @@
+export default class TodoItem {
+  constructor(title, description, dueDate, priority, projectId, status = 0) {
+    this.title = title;
+    this.description = description;
+    this.dueDate = dueDate;
+    this.priority = priority;
+    this.projectId = projectId;
+    this.status = status;
+  }
+}
```

Comment:
Since you used a `Class` structure, you might have well have brought all the code related to `todoItem` inside `src/components/todo.js` inside this class..so that they form a compact/single class-structure

Reply:
Here you using a `Class` as a simple CONSTRUCTOR FUNCTION